src/components/Integrations/Shopify/Shopify.js

```diff
@@ -5,13 +5,21 @@ import { InjectAppServices } from '../../../services/pure-di';
 import logo from './logo.svg';
 import './Shopify.css';
 import { FormattedHTMLMessage, FormattedMessage, injectIntl } from 'react-intl';
+import styled from 'styled-components';
 
 const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
   const [shops, setShops] = useState([]);
   const [isConnected, setIsConnected] = useState(null);
   const [error, setError] = useState(null);
   const [isLoading, setIsLoading] = useState(true);
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const ShopifyLogo = ({ className, svglogo }) => (
```

Comment:
Is `className` parameter required? how does it work?

Reply:
I've read it is neccesary here: https://www.styled-components.com/docs/basics#styling-any-component